cider/exec/module/CiderCompileModule.cpp

```diff
@@ -306,6 +312,52 @@ class CiderCompileModule::Impl {
     return ciderCompilationResult;
   }
 
+  void getArrowMinMaxAndResetNull(void* buf,
+                                  const int64_t row_num,
+                                  const ::substrait::Type& type,
+                                  int64_t* min,
+                                  int64_t* max,
+                                  const int8_t* null_buff) {
+    int64_t* buffer;
+    if (type.has_i64()) {
+      int64_t* buffer = reinterpret_cast<int64_t*>(buf);
+      initColRangeAndResetNull<int64_t>(buffer, row_num, min, max, null_buff);
+    } else if (type.has_i32()) {
+      int32_t* buffer = reinterpret_cast<int32_t*>(buf);
+      initColRangeAndResetNull<int32_t>(buffer, row_num, min, max, null_buff);
+    } else {
+      return;
```

Comment:
is it possible to reach this line? We may can add a line like `CHECK(type.has_i64()||type.has_i32())`.